packages/dx-react-demos/src/bootstrap3/basic/table-row-template.jsx

```diff
@@ -11,21 +11,21 @@ import {
   globalSalesValues,
 } from '../../demo-data/generator';
 
-const TableRow = ({ children, row }) => (
-  <tr
+const TableRow = ({ row, tableRow, ...restProps }) => (
+  <Table.Row
+    {...restProps}
     // eslint-disable-next-line no-alert
     onClick={() => alert(JSON.stringify(row))}
-  >
-    {children}
-  </tr>
+    className={tableRow.rowId % 2 ? 'active' : ''}
```

Comment:
Row id is not the row index. And it is also feels like CSS feature. Can we show a different use case here?

Reply:
Let's discuss it)

Reply:
You may mark rows with a boolean field which means favorite, for example. And highlight such rows with a different styling.

Reply:
I think about the similar scenario based on the 'Sector' field.